platforms/xilinx_vck5000_air/vivado/xilinx_vck5000_air_xsa.tcl

```diff
@@ -64,7 +64,8 @@ if { [string first $scripts_vivado_version $current_vivado_version] == -1 } {
    return 1
 }
 
-set_param board.repoPaths /wrk/xcohdnobkup1/jmelber/vck5000/boards/vck5000/production_silicon/1.1
+#set_param board.repoPaths /wrk/xcohdnobkup1/jmelber/vck5000/boards/vck5000/production_silicon/1.1
+set_param board.repoPaths /group/xrlabs2/eddier/board_files/vck5000/production_silicon/1.1
```

Comment:
We should put these under `/group/xrlabs2/platforms/board_files`

Reply:
Good point, will do.

Reply:
Why aren't these checked in somewhere?

Reply:
They are available on the Early Access Site: [https://www.xilinx.com/member/vck5000.html#vivado](https://www.xilinx.com/member/vck5000.html#vivado)

Reply:
got it. Related question, what happened to the Xilinx BoardStore? https://github.com/Xilinx/XilinxBoardStore/pull/560 What's XHUB?

Reply:
Done in commit f448895